dev/com.ibm.ws.security.openidconnect.clients.common/src/com/ibm/ws/security/openidconnect/clients/common/OidcSessionHelper.java

```diff
@@ -0,0 +1,61 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.security.openidconnect.clients.common;
+
+import org.apache.commons.codec.binary.Base64;
+
+/**
+ * Class used to help generate session id's which contain information
+ * about the oidc config id, sub, sid, and timestamp.
+ * The session id's are stored in the WASOidcSession cookie to keep track of
+ * sessions in the RP.
+ */
+public class OidcSessionHelper {
+    private static String DELIMITER = ",";
```

Comment:
Still thinking we might want a multi-character delimiter to further lower the chances of one of the component strings containing it.

Reply:
i could also try:
`Base64(configId):Base64(sub):Base64(sid):Base64(timestamp)`

don't need to worry about the delimiter clashing this way.

Reply:
I like that. Let's do that 👍